### ISSUE TITLE
Unidecode Error

### ISSUE THREAD

**Issue by Crispae** [pyobo/pyobo#150]
I tried to load HGNC, ontology. I am getting Unidecode error. It is latest version 
![Screenshot 2023-05-09 115836](https://user-images.githubusercontent.com/74324530/237062680-baea14e4-5dfa-4488-a16f-6960c88aaa3d.png)

**Comment by cthoyt** [comment]
Hi @Crispae, I can't replicate your issue. Can you please give more insight into the OS you're using and the PyOBO version you're using? I guess you're on windows, and I wonder if the default character set  is making issues. Maybe on line 226 if the encoding is explicitly set to utf-8, this will work. Can you please try that?

**Comment by Crispae** [comment]
yes, I am using on Window, I changed the encoding. Now its giving different Error.

`~\Anaconda3\lib\site-packages\pyobo\utils\io.py in open_multimap_tsv(path, use_tqdm, has_header)
     80 ) -> Mapping[str, List[str]]:
     81     """Load a mapping TSV file that has multiple mappings for each."""
---> 82     return multidict(_help_multimap_tsv(path=path, use_tqdm=use_tqdm, has_header=has_header))
     83 
     84 

~\Anaconda3\lib\site-packages\pyobo\utils\io.py in multidict(pairs)
    100     """Accumulate a multidict from a list of pairs."""
    101     rv = defaultdict(list)
--> 102     for key, value in pairs:
    103         rv[key].append(value)
    104     return dict(rv)

ValueError: not enough values to unpack (expected 2, got 0)`

**Comment by Crispae** [comment]
still having the same error, with warning


WARNING: [2023-05-09 12:33:43] pyobo.utils.io - [C:\Users\Y-8874903-E.ESTUDIANT\.data\pyobo\raw\so\2021-11-22\cache\names.tsv] malformed row can not be put in dict: []

**Comment by Crispae** [comment]
When I try to use get_filtered_xrefs, I am getting the same error with different databases.

**Comment by Crispae** [comment]
Perfect I will test other ontologies as well, and then I will send you their error logs. Thank You for your assistance

**Comment by cthoyt** [comment]
I'm going to close this, if there are further issues and you can provide more details, please feel free to message back and I'll open it again
